apps/finance/app/src/script.js

```diff
@@ -10,6 +10,23 @@ import vaultBalanceAbi from './abi/vault-balance.json'
 import vaultGetInitializationBlockAbi from './abi/vault-getinitializationblock.json'
 import vaultEventAbi from './abi/vault-events.json'
 
+// Some known tokens don’t strictly follow ERC-20 and it would be difficult to
+// adapt to every situation. The data listed in this map is used as a fallback
+// if either the name or the symbol can’t be determined from the contract only.
+const KNOWN_TOKENS_FALLBACK = new Map([
```

Comment:
I would put this in another file, so we can also use in [`Deposit`](https://github.com/aragon/aragon-apps/blob/master/apps/finance/app/src/components/NewTransfer/Deposit.js#L148).